[SETUP ISSUE] remove_hosts_entries throws sed: -e expression #1, char 1: unknown command: `.'

**Environment**
OS: Ubuntu 18.04
PHP version: unrelated
Installed Codeception unrelated
Installed wp-browser version: 2.2.18
WordPress version: unrelated
Local development environment: docker
WordPress structure and management: docker

Hello, I'm crafting the CI/CD makefile for my plugin which eventually will be running on travis. First I'd like to make it work locally.

I'm getting this error when running `make sync_hosts_entries`:
```
echo "Removing project wp-browser hosts entries..."
Removing project wp-browser hosts entries...
sudo sed -n -i .orig '/## wp-browser project - start ##/{x;d;};1h;1!{x;p;};${x;p;}' /etc/hosts
sed: -e expression #1, char 1: unknown command: `.'
Makefile:78: recipe for target 'remove_hosts_entries' failed
make: *** [remove_hosts_entries] Error 1
```

Do you think it has to do with my environment?

I managed it to work on my local by replacing
```
remove_hosts_entries:
	echo "Removing project ${PROJECT} hosts entries..."
	sudo sed -n -i .orig '/## ${PROJECT} project - start ##/{x;d;};1h;1!{x;p;};$${x;p;}' /etc/hosts
	sudo sed -i .orig '/^## ${PROJECT} project - start ##/,/## ${PROJECT} project - end ##$$/d' /etc/hosts
```
with
```
remove_hosts_entries:
	echo "Removing project ${PROJECT} hosts entries..."
	sudo sed -i '/^## ${PROJECT} project - start ##/,/## ${PROJECT} project - end ##$$/d' /etc/hosts
```

--- a/Makefile
+++ b/Makefile
@@ -250,17 +250,15 @@ gitbook_build: docker/gitbook/id duplicate_gitbook_files module_docs gitbook_ins
 	rm -rf ${CURDIR}/docs/site/bin
 
 remove_hosts_entries:
-	echo "Removing project ${PROJECT} hosts entries..."
-	sudo sed -n -i .orig '/## ${PROJECT} project - start ##/{x;d;};1h;1!{x;p;};$${x;p;}' /etc/hosts
-	sudo sed -i .orig '/^## ${PROJECT} project - start ##/,/## ${PROJECT} project - end ##$$/d' /etc/hosts
+	echo "Removing project ${PROJECT} hosts entries (and backing up /etc/hosts to /etc/hosts.orig...)"
+	sudo sed -i.orig '/^## ${PROJECT} project - Start ##/,/## ${PROJECT} project - End ##$$/d' /etc/hosts
 
 sync_hosts_entries: remove_hosts_entries
 	echo "Adding project ${project} hosts entries..."
 	set -o allexport &&  source .env.testing &&  set +o allexport && \
-	sudo -- sh -c "echo '' >> /etc/hosts" && \
-	sudo -- sh -c "echo '## ${PROJECT} project - start ##' >> /etc/hosts" && \
+	sudo -- sh -c "echo '## ${PROJECT} project - Start ##' >> /etc/hosts" && \
 	sudo -- sh -c "echo '127.0.0.1 $${TEST_HOSTS}' >> /etc/hosts" && \
-	sudo -- sh -c "echo '## ${PROJECT} project - end ##' >> /etc/hosts"
+	sudo -- sh -c "echo '## ${PROJECT} project - End ##' >> /etc/hosts"
 
 # Export a dump of WordPressdatabase to the _data folder of the project.
 wp_dump:

